charts/kargo/README.md

```diff
@@ -145,7 +145,7 @@ the Kargo controller is running.
 | `controller.serviceAccount.iamRole`          | Specifies the ARN of an AWS IAM role to be used by the controller in an IRSA-enabled EKS cluster.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | `""`                |
-| `controller.globalCredentials.namespaces`    | List of namespaces to look for shared credentials.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | `[]`                |
+| `controller.globalCredentials.namespaces`    | List of namespaces to look for shared credentials. Note that as ov v1.0.0, the Kargo controller does not have cluster-wide access to Secrets. The controller receives read-only permission for Secrets on a per-Project basis as Projects are created. If you designate some namespaces as homes for "global" credentials, you will need to manually grant the controller permission to read Secrets in those namespaces.                                                                                                                                                                                                                                                                                                        | `[]`                |
```

Comment:
cc @gdsoumya

Reply:
It learns the namespaces from an env var at startup. So it knows what "global" namespaces _should_ have bindings, but it has no way to know what things might have _previously_ been on that list, removed before being restarted, and now ought to _not_ have those bindings. So we can ensure existence easily. We cannot ensure removal.